e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -371,6 +371,35 @@ export class ExtrinsicHelper {
     );
   }
 
+  public static async getOrCreateSchemaV3(
```

Comment:
Fairly simple, but so much better for all the tests where we just need a schema, but are not working on testing schemas.